corehq/apps/reports/standard/deployments.py

```diff
@@ -98,9 +70,16 @@ def headers(self):
                              sort_type=DTSortType.NUMERIC),
             DataTablesColumn(_("Last Sync"),
                              sort_type=DTSortType.NUMERIC),
-            DataTablesColumn(_("Application (Deployed Version)"),
-                help_text=_("""Displays application version of the last submitted form;
-                            The currently deployed version may be different."""))
+            DataTablesColumn(_("Application"),
+                             help_text=_("Displays application of last submitted form")),
+            DataTablesColumn(_("Application Version"),
+                             help_text=_("""Displays application version of the last submitted form;
+                                         The currently deployed version may be different."""),
```

Comment:
yeah agree is no biggie. it does create some oddities in our translation files like this:
```
#: corehq/apps/reports/standard/deployments.py:102
msgid ""
"Displays application version of the last submitted form;\n"
"                            The currently deployed version may be different."
msgstr ""
```
so is something i tend to avoid, but agree is not the most important thing in the world.